front/src/components/post/CommentItem.vue

```diff
@@ -22,14 +22,25 @@
 </template>
 
 <script>
+import { CHANGE_DATE_FROM_NOW } from "@/utils/moment";
+
 export default {
   name: "CommentItem",
+  data() {
+    return {
+      commentDate: "",
+    };
+  },
   props: {
     comment: {
       type: Object,
       required: true,
     },
   },
+  async mounted() {
+    console.log(this.comment);
+    this.commentDate = CHANGE_DATE_FROM_NOW(this.comment.createdAt);
```

Comment:
plugin을 추가했다면
```suggestion
    this.commentDate = this.$moment(this.comment.createdAt).fromNow();
```
이 방식으로도 호출할 수 있을거예요!